lgc/builder/BuilderImpl.cpp

```diff
@@ -334,6 +335,67 @@ BranchInst *BuilderImpl::createIf(Value *condition, bool wantElse, const Twine &
   return branch;
 }
 
+// A simple memory efficient container that holds up to 64 instructions in a bit vector. It needs two helper data
+// structures: 1. indexInsnMap that maps an instruction to its index in the bit vector and 2. insnChain that maps an
+// index back to an instruction.
+class TinyInsnSet {
+  BitVector bits;
```

Comment:
```suggestion
  BitVector occupiedInstructionSlots;
```

Reply:
I prefer to use bits as bit vector does.